disdrodb/L0/readers/UK/DIVEN.py

```diff
@@ -82,9 +49,9 @@ def reformat_DIVEN_files(file_list, attrs):
             DISDRODB metadata about the station.
         """
         from disdrodb.L0.utils_nc import xr_concat_datasets
-        from disdrodb.L1_proc import get_L1_coords
+        from disdrodb.L0.L0A_processing import get_L1_coords
```

Comment:
`get_L1_coords` does not exist anymore. 
Replace with `from disdrodb.L0.L0B_processing import get_coords`
https://github.com/ltelab/disdrodb/blob/main/disdrodb/L0/L0B_processing.py#L229

Reply:
ok corrected